### PR TITLE
The company publishes content to the customer circle of friends , Upload attached resource

### PR DESCRIPTION
**Pull request by DivinerSheep**
https://developer.work.weixin.qq.com/document/path/95094
The company publishes content to the customer circle of friends
https://developer.work.weixin.qq.com/document/path/95098
Upload attached resource

### CODE REVIEW

File: src/Work/Media/Client.php [hunk 1]
```diff
@@ -100,4 +110,26 @@ public function upload(string $type, string $path)
...
+
+    /**
+     * Upload media
+     *
+     * @return mixed
+     *
+     * @throws \EasyWeChat\Kernel\Exceptions\InvalidConfigException
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function uploadAttachment(string $path, string $media_type, int $attachment_type)
```
Comment (overtrue) [inline src/Work/Media/Client.php:122]: 请使用驼峰命名

File: src/Work/Media/Client.php [hunk 2]
```diff
@@ -84,6 +84,16 @@ public function uploadFile(string $path)
         return $this->upload('file', $path);
     }
 
+    /**
+     * Upload Attachment Resources
+     *
+     * @return mixed
+     */
+    public function uploadAttachmentResources(string $path, string $media_type = 'image', int $attachment_type = 1)
```
Comment (overtrue) [inline src/Work/Media/Client.php:92]: 请使用驼峰命名

File: src/Work/Media/Client.php [hunk 3]
```diff
@@ -89,9 +89,9 @@ public function uploadFile(string $path)
      *
      * @return mixed
      */
-    public function uploadAttachmentResources(string $path, string $media_type = 'image', int $attachment_type = 1)
+    public function UploadAttachmentResources(string $path, string $media_type = 'image', int $attachment_type = 1)
```
Comment (overtrue) [inline src/Work/Media/Client.php:92]: 额，不是这么改哈，驼峰是 camelCase，大驼峰是 CamelCase，我说的是变量名而不是方法名，`$media_type` -> `$mediaType`

**Comment by DivinerSheep** [comment]
正在上传，尴尬了，大佬一出手就知有没有，写的不规范，平时还是太随意了，要加强学习


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "w7corp/easywechat"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月28日(星期五) 中午11:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [w7corp/easywechat] The company publishes content to the customer circle of friends , Upload attached resource (PR #2688)





 
@overtrue commented on this pull request.
 
 
In src/Work/Media/Client.php:
 &gt; @@ -89,9 +89,9 @@ public function uploadFile(string $path)       *       * @return mixed       */ -    public function uploadAttachmentResources(string $path, string $media_type = 'image', int $attachment_type = 1) +    public function UploadAttachmentResources(string $path, string $media_type = 'image', int $attachment_type = 1)  
额，不是这么改哈，驼峰是 camelCase，大驼峰是 CamelCase，我说的是变量名而不是方法名，$media_type -&gt; $mediaType

**Pull request merged**